### DISCUSSION TITLE
Amazon "Campaign" Sponsored brand v2 deprecated

### DISCUSSION THREAD

**Discussion by michelexport** [amzn/ads-advanced-tools-docs#80] [General]
Hi all,
we are interested in retrieving "cost" for amazon sponsored brand advertising campaign, report "Campaign"
v2 has been deprecated since march 30th trhough.
we looked at v3 but noticed that "Campaign" report is not available for sponsored brand - only "Purchased product report" is available but it does not provide any "cost" related field https://advertising.amazon.com/API/docs/en-us/reporting/v3/report-types
Bottom line: how do you retrieve "cost" for sponsored brand now?

**Comment by elizafarley** [comment]
Hi @michelexport, only Sponsored Products version 2 reporting is being deprecated at this time, so you can continue to use the [V2 campaigns report](https://advertising.amazon.com/API/docs/en-us/reporting/v2/report-types#campaign-reports) for Sponsored Brands at this time.

**Reply by michelexport** [reply]
oh thanks a lot! in our power bi i haven't seen any datas since march 31st, maybe is our internal error then?

**Reply by elizafarley** [reply]
If you are able, try taking a look in the advertising console to see if you have data since March 31st. If you are seeing data in the UI, but not through the API, please [open a JIRA ticket](https://amzn-clicks.atlassian.net/servicedesk/customer/portal/2/group/2) so our engineering team can investigate.
